wiring/inc/spark_wiring_cloud.h

```diff
@@ -78,8 +78,8 @@ class CloudDisconnectOptions {
     bool graceful() const;
     bool isGracefulSet() const;
 
-    CloudDisconnectOptions& timeout(unsigned timeout);
-    unsigned timeout() const;
+    CloudDisconnectOptions& timeout(system_tick_t timeout);
```

Comment:
Add chrono literals overload as well everywhere `system_tick_t` can be taken.

Reply:
:+1: